app/scripts/directives/logViewer.js

```diff
@@ -434,16 +436,21 @@ angular.module('openshiftConsole')
                 if(!(projectName && containerName && name && url)) {
                   return;
                 }
-                $scope.$watchGroup(['context.project.metadata.name', 'options.container', 'name'], function() {
-                  angular.extend($scope, {
-                    kibanaArchiveUrl: $sce.trustAsResourceUrl(logLinks.archiveUri({
-                                        baseURL: url,
-                                        namespace: $scope.context.project.metadata.name,
-                                        namespaceUid: $scope.context.project.metadata.uid,
-                                        podname: name,
-                                        containername: $scope.options.container,
-                                        backlink: URI.encode($window.location.href)
-                                      }, $filter('annotation')($scope.context.project,'loggingDataPrefix')))
+
+                AggregatedLoggingService.isOperationsUser().then(function(canViewOperationsLogs) {
+                  $scope.$watchGroup(['context.project.metadata.name', 'options.container', 'name'], function() {
+                    angular.extend($scope, {
+                      // The archive URL violates angular's built in same origin policy.
+                      // Need to explicitly tell it to trust this location or it will throw errors.
```

Comment:
I'm not sure if this is still needed now that it's a link instead of a form. You can probably get rid of the `trustAsResourceUrl` call

Reply:
Comment is out of date with the code now